avb/utils/datasets.py

```diff
@@ -22,7 +22,60 @@
 np.random.seed(config['seed'])
 
 
-def load_svhn(local_data_path=None, one_hot=True, grayscale=True):
+def load_usps(local_data_path=None):
```

Comment:
This is the reStructured docstring style and the used so far is Google. If you are using PyCharm you can set the style in Settings>Tools>Python Integrated Tools and there dosctring format: Google. 
It is not that important though, so leave it at the end if you have other things to do.